src/components/pages/StudentHome/StudentSidebar.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Menu, Layout } from 'antd';
+import 'antd/dist/antd.css';
+import {
+  DashboardOutlined,
+  CheckCircleOutlined,
+  MessageOutlined,
+  GroupOutlined,
+  StarOutlined,
+  SlidersOutlined,
+  ExportOutlined,
+  ReadOutlined,
+} from '@ant-design/icons';
+import { useOktaAuth } from '@okta/okta-react';
+
+const { Sider } = Layout;
+
+const StudentSideBar = props => {
+  const [collapsed, setCollapsed] = useState(false);
```

Comment:
Idea is to have Hamburger menu, so this part of AntD is not going to be used.. nor the collaps. properties in <Sidebar>

Reply:
We'll want to consider what Uros is saying here and just ensure that we can slot this dashboard into the appropriate place successfully.

Reply:
This was mirrored from the Parent Sidebar. @ivanovicuros are you saying there is an entirely new design for the sidebar?

Reply:
@ivanovicuros I would love this context too when you get a chance!